pkg/asset/installconfig/password.go

```diff
@@ -0,0 +1,63 @@
+package installconfig
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	"github.com/openshift/installer/pkg/asset"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var (
+	passwordPrompt = fmt.Sprintf("Enter password (6-16 characters):")
```

Comment:
Where does this restriction come from?

Reply:
Nowhere really. Just wanted to put a finite limit. Should I remove this?
I feel there should be some upperbound though.

Reply:
> Should I remove this?

Please :).

> I feel there should be some upperbound though.

If our backend images (RHCOS?) have an upper bound, I'd be fine validating against it.  But as far as I'm aware, they don't.  If the images don't care, why would the installer?